unitTests/magicbox/SuggestionsManagerTest.ts

```diff
@@ -286,10 +286,12 @@ export function SuggestionsManagerTest() {
         return (magicBoxContainer = $$('div'));
       }
 
+      let blurSpy: jasmine.Spy;
```

Comment:
Instead of creating a `describe` block scoped variable to hold the blur spy, it would be much cleaner to store the mock InputManger. If we reinit it in the `beforeEach`, then we can also remove the need to reset the spy.

```
let inputManager: InputManager;

beforeEach(() => {
    inputManager = mockInputManager();
    suggestionsManager = new SuggestionsManager(..., inputManager, ...);
})
```

Reply:
The spy was getting reset because it was tested multiple times in the same test, but using `toHaveBeenCalledTimes(i)` would remove the need for it. I'll remove the `blurSpy` variable and store the mock instead.